src/test/java/org/mockitousage/serialization/DeepStubsSerializableTest.java

```diff
@@ -47,6 +51,8 @@ public void should_serialize_and_deserialize_parameterized_class_mocked_with_dee
 
     @Test(expected = ClassCastException.class)
     public void should_discard_generics_metadata_when_serialized_then_disabling_deep_stubs_with_generics() throws Exception {
```

Comment:
Let's add a reverse test that shows developer experience when we're running on Java9 and try to use serializable deep stubs. This will cover the exception handling logic.

Reply:
This is obsolete as serialization works again as it did before.